rllib/evaluation/tests/test_envs_that_crash.py

```diff
@@ -108,7 +108,7 @@ def test_crash_only_one_worker_during_sampling_but_recreate(self):
                 # Re-create failed workers (then continue).
                 recreate_failed_workers=True,
             )
-            .training(train_batch_size=60)
```

Comment:
Can you help me understand what changed that these parameters are required now?

Reply:
We changed from PG to PPO, which has specific requirements for batch size and mini batch size (the latter must not be larger than the former). Hence the failure (config validation failed).

Reply:
This makes sense.